graph.go

```diff
@@ -130,9 +135,13 @@ func Graph(units unit.SourceUnits) (*graph.Output, error) {
 		pkgs = append(pkgs, pkg)
 	}
 
-	o, err := doGraph(pkgs)
-	if err != nil {
-		return nil, err
+	var o gog.Output
+	for _, pkg := range pkgs {
+		output, err := doGraph(pkg)
+		if err != nil {
+			return nil, err
+		}
+		o.Append(output)
 	}
```

Comment:
Not saying we should do it now, but as a perf optimisation would it be possible to do this in parallel? Is that worth it?